### PR TITLE
add delete post functionality, update errors file and news feed type …

### PR DESCRIPTION
**Pull request by Siiddy**
add delete post functionality, update errors file and news feed type conversation in locale storage.

### CODE REVIEW

File: src/app/components/business/DeleteModalPost.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          </button>
+          {error && <div className="text-red-500 mt-2">{error}</div>}
+        </div>
+      ) : (
+        <div className="text-center p-4">
+          {toast.message && (
+            <div className={`${toast.isSuccess ? "text-green-500" : "text-red-500"} mb-2`}>
+              <CheckIcon className="h-10 w-10 text-green-500 mx-auto" />
+              {toast.message}
+            </div>
```
Comment (lchaufournais) [inline src/app/components/business/DeleteModalPost.jsx]: use component Toast

**Review by lchaufournais — CHANGES_REQUESTED** [review]

File: src/api-post/src/routes/posts/DeletePostRoute.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          success: false,
+          message: `${HTTP_ERRORS.DELETE_POST_FAILED}: ${error.message}`
+        },
+        HTTP_STATUS_CODES.INTERNAL_SERVER_ERROR
+      )
+    }
+  })
+
+  app.route("/", DeletePostData)
+}
```
Comment (avetisk) [inline src/api-post/src/routes/posts/DeletePostRoute.js:54]: Helpers~!

File: src/app/components/business/DeleteModalPost.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  const handleDelete = async () => {
+    try {
+      await deletePost({ userId: session.id, postId })
+      setIsDeleted(true)
+      showToast("Post deleted successfully! Please wait a few moments...", true)
+      setTimeout(() => {
+        router.reload()
+      }, 2000)
```
Comment (avetisk) [inline src/app/components/business/DeleteModalPost.jsx:31]: Store the duration in a config file and use it everywhere you need such duration.

Would make your app much more homogenous.

File: src/pages/home/index.jsx [hunk 4]
```diff
@@ -11,7 +11,11 @@ const Home = () => {
   //   action: { getImagesBucket }
   // } = useAppContext()
```
Comment (avetisk) [inline src/pages/home/index.jsx:12]: Dead code 😱

**Pull request merged**